tests/cancun/eip6780_selfdestruct/test_selfdestruct.py

```diff
@@ -693,14 +693,12 @@ def test_selfdestruct_pre_existing(
         - Different send-all recipient addresses: single, multiple, including self
         - Different initial balances for the self-destructing contract
     """
-    selfdestruct_contract_address = pre.deploy_contract(selfdestruct_code)
+    selfdestruct_contract_address = pre.deploy_contract(selfdestruct_code, balance=selfdestruct_contract_initial_balance)
```

Comment:
This line is tripping the linting because is longer than 99 characters, should be a matter of splitting into two lines and we should be able to merge afterwards.

Reply:
```
lint: commands[0]> ruff check --no-fix --show-fixes src tests docs .github/scripts
All checks passed!
lint: commands[1]> ruff format --diff src tests docs .github/scripts
441 files already formatted
  lint: OK (0.19=setup[0.09]+cmd[0.07,0.03] seconds)
  congratulations :) (0.24 seconds)
```
Run again, seem good.